falcon/request.py

```diff
@@ -340,9 +340,12 @@ def date(self):
     @property
     def range(self):
         try:
-            value = self.env['HTTP_RANGE']
+            value = self.env['HTTP_RANGE'].strip()
```

Comment:
Web servers typically take care of stripping whitespace from header values, so I don't think this is necessary.